### ISSUE TITLE
Extra ) ?

### ISSUE THREAD

**Issue by symwell** [django-oscar/django-oscar#4052]
Found a bug? Please fill out the sections below.

### Issue Summary

Is there an extra parenthesis [here](https://github.com/django-oscar/django-oscar/blob/master/src/oscar/apps/catalogue/reviews/apps.py#L26)?

Should
```
 path('<int:pk>)/vote/', login_required(self.vote_view.as_view()), name='reviews-vote'),
```
be
```
 path('<int:pk>/vote/', login_required(self.vote_view.as_view()), name='reviews-vote'),
```
?

**Comment by specialunderwear** [comment]
yes I think so, most likely at the moment there is a ) in the url for reviews. Could you send a PR with a fix?

**Comment by WaqarAnwar** [comment]
Hello, I have opened a PR with the fix. Please have a look. Thanks.

**Comment by joeyjurjens** [comment]
Fixed in #4108
